.github/workflows/push.yml

```diff
@@ -66,3 +67,29 @@ jobs:
         python setup.py sdist bdist_wheel
         twine check dist/*
         twine upload dist/*
+
+  update-mycroft-core:
+    runs-on: ubuntu-latest
+    needs: publish
+    steps:
+    - uses: actions/checkout@v2
+      with: 
+        repository: MycroftAI/mycroft-core
+
+    - name: Update mycroft-core
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        git config user.name mycroft-devs
+        git config user.email devs@mycroft.ai
+        sed "s/adapt-parser==[0-9].[0-9].[0-9]/adapt-parser==$VERSION/" requirements/requirements.txt > tmp-requirements.txt
```

Comment:
Per recommendations from [semver.org](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string), this is not the correct format. I believe each segment needs to be of the form `[0-9]+` to match correctly.

Reply:
Yeah good call - I also hadn't escaped the period characters so this may have ended up with all sorts of weird.

I did the simplified version of the regex as you suggested `[0-9]+`. It looks like the recommended version is trying to prevent matching on things like 1.02.03 but this is easier to read and seems fine to me.

It also made me think that this would operate on pre-release versions which is probably not what we want. So I've added a quick conditional to check for a hyphen which I believe should only be present if it's a pre-release version.